docs/bengali.rst

```diff
@@ -15,3 +18,22 @@ Use ``CorpusImporter()`` or browse the `CLTK GitHub organization <https://github
    In [3]: c.list_corpora
    Out[3]:
    ['bengali_text_wikisource']
+
+Tokenizer
+=========
+
+This tool can help break up a sentence into smaller constituents. 
+
+.. code-block:: python
+
+   In [1]: from cltk.tokenize.indian_tokenizer import indian_punctuation_tokenize_regex as i_word
+
+   In [2]: sentence = "রাজপণ্ডিত হব মনে আশা করে | সপ্তশ্লোক ভেটিলাম রাজা গৌড়েশ্বরে ||"
```

Comment:
@djokester Isn't this the same modern Bengali sentence as in the previous submission? Just find a example of Old Bengali and replace.

Also, don't worry about what the regression tests say about non-code submissions. If there is a problem, I will let you know.